crawl-ref/source/dat/des/branches/lair.des

```diff
@@ -2467,6 +2485,198 @@ PwwwwwwP;-+---;x".""""xx
+    ..J...J..
+   ..QDDDDDR..
+  ..QwQ'''RwR..
+ ..QwQ'''''RwR..
+..QwQ'''''''RwR..
+.JwQ'''''''''RwJ.
+.JwJ'''HHH'''JwJ.
+.JwJ'''HHH'''JwJ.
+.JwJ'''HHH'''JwJ.
+.JwZ'''''''''SwJ.
+..ZwZ'''''''SwS..
+ ..ZwZ'''''SwS..
+  ..ZwZJJJSwS..
+   ..ZwwwwwS..
+    ..JJJJJ..
+     .......
+ENDMAP
+
+NAME:   nicolae_sub_lair_graze_edge
+TAGS:   allow_dup nico_grazesub no_pool_fixup unrand
+KMONS:  1 = plant
+KMONS:  2 = bush
+KFEAT:  12 = W / .
+NSUBST: D = w / J, E = w / J, F = w / J, H = w / J
+SUBST:  J = .:20 W:20 1:20 2w, w = wt
+MAP
+ D
+D E
+ E F
+  F H
+   H
+ENDMAP
+
+NAME:  nicolae_lair_lakeside_cave
+TAGS:  no_pool_fixup transparent
+DEPTH: Lair
+KITEM: DEF = % w:40 / * w:16 / | w:4 / nothing w:20
+KMONS: 1D = alligator / anaconda / manticore / fire crab / polar bear / \
+            torpor snail / komodo dragon / dire elephant
+KMONS: 2E = hornet / hell rat / wyvern / cane toad / death yak / elephant / \
+            steam dragon / rime drake / hydra
+KMONS: 3F = basilisk / scorpion / wolf / black mamba / water moccasin / \
+            black bear / yak / bullfrog
+SUBST: K = xxw..., M = wW, N = WWwx, - = x_, H = D E:20 F:30, \
+       p = 2:5 3:80 .:180, q = 1 3 2:80 .:140
+CLEAR: _
+MAP
+                 @..Mw--
+ -----------     ...Mww--
+--xxxxxxxxxxx    ...Mwww--
+-xxxKKKKKKKKxx    ...Mwwx-
+-xxKKqqq.ppKKxx    ...Mwx-
+-xKKqqqMMpppKxxxxxxxppMwx-
+-xKqqqMMMMppppppxxppppMwx-
+-xKqqMwwwwMMMppppppppMwwx-
+-xKqMMwwwwwwwMMMppMMMwwwx-
+-xKqMMwwwwwwwwwwMMwwwwwxx-
+-xKqqMwwwwMxxxwwwwwwxxxx--
+-xKqqqMMMM.+.xxxxxxxxxxx-
+-xKKqqqMM.1x..HHHDxxxxx--
+-xxKKqqq.1.xN.HHHDxxxx--
+--xxKKKKKKxxNNNNxxxx---
+ --xxxxxxxxxxxxxxx---
+  -----------------
+ENDMAP
+
+NAME:   nicolae_lair_lil_pond
+TAGS:   transparent ruin decor extra
+DEPTH:  Lair
+NSUBST: D = . / x
+SUBST:  - = . / W
+MAP
+............
+....xxxx....
+..DDDxxxxx..
+..x------x..
+.xx-wwww-xx.
+.xx-wwww-xx.
+.xx-wwww-xx.
+.xx-wwww-xx.
+..x------x..
+..xxxxxxxx..
+....xxxx....
+............
+ENDMAP
+
+NAME:  nicolae_lair_lindhole
+TAGS:  transparent
+DEPTH: Lair
+KMONS: 1 = lindwurm
+SUBST: K = LLLL., L = lx, ~ = 1.., - = x_, % = %%%$*.
+CLEAR: _
+MAP
+-----------
+-KKKKKKKKKK
+-K....~...@
+-K.~.......
+-L..LLLLLLL-
+-L..Lxxxxxx-
+-L..LLLLLLx-
+-L.~...%%Lx-
+-L....1%%Lx-
+-LLLLLLLLLx-
+-xxxxxxxxxx-
+------------
+ENDMAP
+
+NAME:  nicolae_lair_liquid_snakes
+TAGS:  no_pool_fixup transparent
+DEPTH: Lair
+KMONS: 1 = lava snake
+KMONS: 2 = electric eel
+KFEAT: 1 = l
+KFEAT: 2 = w
+SUBST: L = ..D, M = ..E, - = xx_, D = 1 l:240, E = 2 w:240
+CLEAR: _
+MAP
+        -----------
+       --xxxxxxxxx--
+--------xxDDD1DDDxx-
+xxxxxxxxxDDLLLLLDDxx
+xDDDDxxxDDLL...LLDDx
+x.LLDDxDDLL.....LL.x
+@..LLD1DLL.MMMMM...@
+@...LLLLL.MME2EMM..@
+x.MM.....MMEExEEMM.x
+xEEMM...MMEExxxEEEEx
+xxEEMMMMMEExxxxxxxxx
+-xxEEE2EEExx--------
+--xxxxxxxxx--
+ -----------
+ENDMAP
+
+NAME:   nicolae_lair_run_with_the_wolves
+TAGS:   ruin transparent
+DEPTH:  Lair
+KMONS:  1 = wolf spider
+KMONS:  2 = wolf band
+NSUBST: D = 1 / 12 / 2
```

Comment:
I was going to say that two wolf spiders might be a bit much for early lair, but there's apparently already a lair vault (venomous_thicket) that can place more than that, albeit rarely. Funny concept.